examples/SampleUtils.cpp

```diff
@@ -81,6 +81,75 @@ namespace utils {
+            std::memcpy(minConstant->data(), &min, sizeof(float));
+            constants.push_back(minConstant);
+            float max = 6;
+            auto maxConstant = std::make_shared<std::vector<char>>(sizeof(float));
+            std::memcpy(maxConstant->data(), &max, sizeof(float));
+            constants.push_back(maxConstant);
+            clampOptions.minValue =
+                utils::BuildConstant(builder, {}, minConstant->data(), sizeof(float));
+            clampOptions.maxValue =
+                utils::BuildConstant(builder, {}, maxConstant->data(), sizeof(float));
+            activationOperator = builder.ClampOperator(&clampOptions);
+        } else if (activation == FusedActivation::SIGMOID) {
+            activationOperator = builder.SigmoidOperator();
+        } else if (activation == FusedActivation::LEAKYRELU) {
+            ml::LeakyReluOptions leakyReluOptions;
+            leakyReluOptions.alpha = options[0];
+            activationOperator = builder.LeakyReluOperator(&leakyReluOptions);
+        } else {
+            assert(0);
+        }
+        return activationOperator;
+    }
+
+    const ml::Operand createActivationOperand(std::vector<SHARED_DATA_TYPE>& constants,
+                                              const ml::GraphBuilder& builder,
+                                              const ml::Operand& input,
+                                              FusedActivation activation,
+                                              const std::vector<float>& options) {
+        ml::Operand activationOperand;
+        if (activation == FusedActivation::RELU) {
+            activationOperand = builder.Relu(input);
+        } else if (activation == FusedActivation::RELU6) {
+            ml::ClampOptions clampOptions;
+            float min = 0;
+            auto minConstant = std::make_shared<std::vector<char>>(sizeof(float));
+            std::memcpy(minConstant->data(), &min, sizeof(float));
+            constants.push_back(minConstant);
+            float max = 6;
+            auto maxConstant = std::make_shared<std::vector<char>>(sizeof(float));
+            std::memcpy(maxConstant->data(), &max, sizeof(float));
+            constants.push_back(maxConstant);
+            clampOptions.minValue =
+                utils::BuildConstant(builder, {}, minConstant->data(), sizeof(float));
+            clampOptions.maxValue =
+                utils::BuildConstant(builder, {}, maxConstant->data(), sizeof(float));
+            activationOperand = builder.Clamp(input, &clampOptions);
+        } else if (activation == FusedActivation::SIGMOID) {
+            activationOperand = builder.Sigmoid(input);
+        } else if (activation == FusedActivation::LEAKYRELU) {
+            ml::LeakyReluOptions leakyReluOptions;
+            leakyReluOptions.alpha = options[0];
+            activationOperand = builder.LeakyRelu(input, &leakyReluOptions);
+        } else {
+            assert(0);
```

Comment:
`assert` would be removed in release build. Please use `DAWN_ASSERT`.

Reply:
Thanks, got it.